src/atoms/hooks/usePaneActiveEditorContentAtoms.ts

```diff
@@ -0,0 +1,29 @@
+import { atom, useAtomValue } from 'jotai';
+import { selectAtom } from 'jotai/utils';
+import { useMemo } from 'react';
+
+import { editorsContentStateAtom } from '../core/editorContent';
+import { EditorContentAtoms } from '../types/EditorContentAtoms';
+import { PaneId } from '../types/PaneGroup';
+import { usePaneActiveEditorId } from './usePaneActiveEditorId';
+
+/** Returns the the content of the active editor of the given pane */
+export function usePaneActiveEditorContentAtoms(paneId: PaneId): EditorContentAtoms | null {
```

Comment:
As discussed offline, we will not adopt this convention, to avoid having names that are too long. But we will remove `Pane` and add `ForPane` at the end, like `useActiveEditorContentAtomsForPane` for this hook